cmd/google_cloud_ops_agent_engine/main.go

```diff
@@ -65,7 +63,13 @@ func run() error {
 	log.Printf("Built-in config:\n%s", apps.BuiltInConfStructs["linux"])
 	log.Printf("Merged config:\n%s", uc)
 
-	runStartupChecks(*service)
+	if *service == "" {
+		runHealthChecks(healthchecks.HealthCheckRegistryFactory())
+		log.Println("Startup checks finished")
```

Comment:
Some suggestions for this : 
- Moving the execution of the `healthchecks` to the beginning of `run()` so it's the first thing. The reasoning for this is if `--healthchecks` is set, then it will only run the `healthchecks` and no other part of the confgenerator.
- I think there's no need anymore to pass as a paramter `healthchecks.HealthCheckRegistryFactory()`.
- Maybe add a comment to the `return nil` explaining why it's set this way.